app/views/manage_assignments/coverages/_coverage.html.erb

```diff
@@ -13,6 +13,9 @@
     </h2>
     <p class="class-card-heading-outcomes">
       <%= t(".matched_outcomes", count: coverage.outcome_coverages.size) %>
+    <p>
```

Comment:
@mborsare, You are correct the user can only attach one item at the class level.